frontend/app/templates/components/pricing-component.hbs

```diff
@@ -6,43 +6,60 @@
         <p class="heading-alpha">Pricing</p>
         <br>
         <div class="inline-columns">
-            <div class="ui stackable two column grid container center aligned">
+            <div class="ui stackable three column grid container center aligned">
```

Comment:
Use class `unstackable`, To have responsiveness in mobile view.

Reply:
Alright